scaleway/resource_instance_ip_reverse_dns.go

```diff
@@ -18,6 +18,9 @@ func resourceScalewayInstanceIPReverseDNS() *schema.Resource {
 		Importer: &schema.ResourceImporter{
 			StateContext: schema.ImportStatePassthroughContext,
 		},
+		Timeouts: &schema.ResourceTimeout{
+			Default: schema.DefaultTimeout(instanceIPTimeout),
```

Comment:
I wounder if we would not be easier to read if time was directly specified without using a varibale

Reply:
I would prefer to have all timeout for a given set of resources be defined in a single place that allows for easy modification and reading once when we want to change the default for a given set of resources.